<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sourabhdh-w7" name="Tests []" tests="2" failures="1" timestamp="25 Nov 2016 12:45:57 GMT" time="502.329" errors="0">
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="com.test.AdminTestSample">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ideator.util.PageClass.teardown(PageClass.java:124)
	at com.test.AdminTestSample.afterMethod(AdminTestSample.java:59)
	at com.test.IdeatorStart.testNgRun(IdeatorStart.java:49)
	at com.test.IdeatorStart.main(IdeatorStart.java:114)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="com.test.AdminTestSample">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.test.AdminTestSample">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="addUserCommunity" time="10.303" classname="com.test.AdminTestSample">
    <failure type="java.lang.AssertionError" message="expected [http://ucsdtest.ideator.com/] but found [https://ucsdtest.ideator.com/ideas?after_login=true]">
      <![CDATA[java.lang.AssertionError: expected [http://ucsdtest.ideator.com/] but found [https://ucsdtest.ideator.com/ideas?after_login=true]
	at com.test.AdminTestSample.addUserCommunity(AdminTestSample.java:37)
	at com.test.IdeatorStart.testNgRun(IdeatorStart.java:49)
	at com.test.IdeatorStart.main(IdeatorStart.java:114)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- addUserCommunity -->
  <testcase name="createNewIdea" time="0.0" classname="com.test.AdminTestSample">
    <skipped/>
  </testcase> <!-- createNewIdea -->
</testsuite> <!-- Tests [] -->
